src/utils/richObjects.ts

```diff
@@ -46,6 +46,17 @@ export function mapRichObjectToRichArgument(richObject: IRichObject) {
 			props: richObject,
 		}
 	case 'user':
+		if (richObject.server) {
+			return {
+				component: NcUserBubble,
+				props: {
+					avatarImage: 'icon-user',
+					displayName: richObject.name,
+					user: richObject.id,
+					url: richObject.link,
+				},
+			}
+		}
```

Comment:
Maybe replace existing code with it to avoid duplications?
```suggestion
		return {
			component: NcUserBubble,
			props: {
				// show thumbnail for federated users
				avatarImage: richObject.server ? 'icon-user' : undefined,
				displayName: richObject.name,
				user: richObject.id,
				url: richObject.link,
			},
		}
```